garrysmod/resource/localization/en/spawnmenu.properties

```diff
@@ -166,4 +166,20 @@ spawnmenu.seat.jalopy=Jalopy Seat
 spawnmenu.seat.airboat=Airboat Seat
 spawnmenu.seat.simple_sit=Car Seat (Sit)
 spawnmenu.seat.simple_jeep=Car Seat (Jeep)
-spawnmenu.seat.simple_airboat=Car Seat (Airboat)
\ No newline at end of file
+spawnmenu.seat.simple_airboat=Car Seat (Airboat)
+
+# Tabs categories
+# Entities
+spawnmenu.ents.category.funandgames=Fun + Games
+spawnmenu.ents.category.editors=Editors
+# NPCs
+spawnmenu.npcs.category.humans_and_resistance=Humans + Resistance
+spawnmenu.npcs.category.zombie_and_enemy_aliens=Zombies + Enemy Aliens
+spawnmenu.npcs.category.animals=Animals
+spawnmenu.npcs.category.combine=Combine
+spawnmenu.npcs.category.nextbot=Nextbot
+# Vehicles
+spawnmenu.vehicles.category.chairs=Chairs
```

Comment:
> These should follow the same string format as categories on lines 11-15, i.e. `spawnmenu.category.X`.
> 
> Also the way you use `and` is inconsistent, and probably should be dropped entirely. The internal strings do not need to match the English translation, they should be short.
> 
> So `spawnmenu.ents.category.funandgames` => `spawnmenu.category.fun_games` `spawnmenu.npcs.category.zombie_and_enemy_aliens` => `spawnmenu.category.zombie_aliens` etc.

Now I understand how it should look.

Wouldn't it be better to use `spawnmenu.section.X` or `spawnmenu.subcategory.X` then?

`spawnmenu.category.X` used for tabs in the spawnmenu, rather than actual categories, so I believe it should be different anyway. (or rename current `spawnmenu.category.X` strings => `spawnmenu.tabs.X` while those above will take `spawnmenu.category.X`)